config/routes.rb

```diff
@@ -116,6 +116,10 @@
     member do
       get :schedule
     end
+
+    member do
+      get :events
```

Comment:
Any reason why this is in a new `member do` block and not added on the one above?